platform/src/test/java/org/exoplatform/platform/qa/ui/platform/social/SOCNotificationsIntranetNotificationTypesTestIT.java

```diff
@@ -0,0 +1,3490 @@
+    manageLogInOut.signIn(username5, password);
+    homePagePlatform.goToConnections();
+    connectionsManagement.acceptAConnection(username1);
+    homePagePlatform.goToHomePage();
+    activityStream.likeActivity(activity);
+
+    info("Check Like notification in intranet notification");
+    manageLogInOut.signIn(username1, password);
+    ELEMENT_ALERT_NOTIFICATION.waitUntil(Condition.visible, Configuration.timeout);
+    ELEMENT_ALERT_NOTIFICATION.click();
+    $(ELEMENT_NOTIFICATION_DROPDOWN).find(byText(activity)).parent().parent().shouldHave(text(username5 + " " + username5 + ", "
+        + username4 + " " + username4 + " and 2 more like your activity."));
+
+    info("Check comment notification in activity Viewer");
+    $(ELEMENT_NOTIFICATION_DROPDOWN).find(byText(activity)).click();
+    $(byText(activity)).parent().parent().parent().find(ELEMENT_ICON_LIKE_ACTIVITY).parent().shouldHave(text("4"));
+    navigationToolbar.goToIntranetNotification();
+    intranetNotification.goToAllNotification();
+    $(byText(activity)).parent().parent().parent().find(byText(username2 + " " + username2)).should(Condition.exist);
+    manageLogInOut.signIn(DATA_USER1, "gtngtn");
+    navigationToolbar.goToManageCommunity();
+    addUsers.deleteUser(username1);
+    addUsers.deleteUser(username2);
+    addUsers.deleteUser(username3);
+    addUsers.deleteUser(username4);
+    addUsers.deleteUser(username5);
+  }
+
+  /**
+   * <li>Case ID:123002.</li>
+   * <li>Test Case Name: Like Notification when a new like is pushed.</li>
+   * Precondition: - User A, User B, User C are connected - User A has posted an
+   * activity - User B has liked User A activity - The notification "Someone likes
+   * one of my activities" is activated in the user settings Step Number: 1 Step
+   * Name: Step 1: Check notifications list Step Description: - Login with User A
+   * - Click the notifications icon in the top navigation - Check the notification
+   * list (keep the notification unread) Input Data: Expected Outcome: - A Like
+   * notification is displayed in the list. Step Number: 2 Step Name: Step 2: Push
+   * a new like notification Step Description: - With User C, like the activity of
+   * User A - Check the notifications list Input Data: Expected Outcome: - The
+   * Like notification is listed/merged in the same previous notification (step 1)
+   * - The notification is displayed at the top of the list. Step Number: 3 Step
+   * Name: Step 3: check the message of the Like notification Step Description: -
+   * Check notification message in the notifications list and View All page Input
+   * Data: Expected Outcome: - The notification message is : LAST_AVATAR
+   * $USER_LIST like your activity $ACTIVITY $DATE Where : - $LAST_AVATAR is the
+   * thumbnail of User C - $USER_LIST is User B, User C - $ACTIVITY is the
+   * activity message/title - $DATE is the date of the last notification of User C
+   */
+  @Test
+  @BugInPLF("PLF-7656")
```

Comment:
This issue has been fixed. To be removed.

Reply:
done